### DISCUSSION TITLE
Problem with VS2022?

### DISCUSSION THREAD

**Discussion by fredericoregateiro** [MahApps/MahApps.Metro#4219] [Q&A] [answered]
Using the VS2019 there is no problem, but when using the same project in VS2022 i get errors like these on the styles.

![error](https://user-images.githubusercontent.com/6304701/144281308-3c2796db-76c7-4e4f-b864-d78c0c6577e4.jpg)

**Comment by seba30** [comment]
Same here.

**Comment by punker76** [comment]
Which VS 2022 version is this? I have v17.0.1 and don't see this.

![image](https://user-images.githubusercontent.com/658431/144454204-0954aacb-6e1a-476e-b3e5-8442bbcc8260.png)

![image](https://user-images.githubusercontent.com/658431/144454440-10f89c25-80e8-4224-b53a-3dd8428e2a72.png)

**Reply by fredericoregateiro** [reply]
I'm using 
Microsoft Visual Studio Community 2022 (64-bit) 
Version 17.0.1
.NET 4.8 Project

**Comment by punker76** [comment, marked as answer]
v17.0.2 works too on my machine™

**Reply by fredericoregateiro** [reply]
Shame on me 🤣 

When trying to build a small sample with this error, i found the error

`
Error	XDG0062	'DataGridNumericUpDownColumn' type must derive from FrameworkElement or FrameworkContentElement.
`
I was defining the styles on the App.xaml 
```xml
<Style TargetType="mah:DataGridNumericUpDownColumn">
    <Setter Property="FontSize" Value="18"/>
    <Setter Property="Culture" Value="pt-PT"/>
    <Setter Property="DecimalPointCorrection" Value="Number"/>
    <Setter Property="HideUpDownButtons" Value="True"/>
</Style>
```

And this was making ALL MahApps references in my styles to go crazy

This only shows on VS2022, on VS2019 works fine.....
The problem with xaml, we make one error, we get 1000 build errors
